src/coreclr/tools/Common/Compiler/DependencyAnalysis/ShadowGenericMethodNode.cs

```diff
@@ -13,21 +13,21 @@
 namespace ILCompiler.DependencyAnalysis
 {
     /// <summary>
-    /// Represents a concrete method on a generic type (or a generic method) that doesn't
+    /// Represents a generic method on a generic type (or a generic method) that doesn't
```

Comment:
"generic method on a generic type (or a generic method)".

The new name of this node is problematic for the reasons seen in this comment - when we say "generic method" in the compiler, we mean a method with `HasInstantiation==true`. This node models methods with some genericness, but not just generic methods.